### ISSUE TITLE
Downloading description as txt

### ISSUE THREAD

**Issue by Spookshow562** [adripo/thingiverse-plus#4]
Could you add ability to download the description and such so that we can save the creators instructions and print settings and such.

**Comment by Valerious-Black** [comment]
+1 on this, it would be incredibly useful

**Comment by h04** [comment]
Yes - I would love this feature!  All of the relevant printing information is contained within this text.
